data/json/items/gunmod/conversions.json

```diff
@@ -397,6 +397,37 @@
     "min_skills": [ [ "weapon", 2 ] ],
     "add_mod": [ [ "brass catcher", 1 ], [ "bayonet lug", 1 ], [ "rail", 2 ], [ "sights", 1 ], [ "underbarrel", 1 ], [ "muzzle", 1 ] ]
   },
+  {
+    "id": "retool_ar15_22",
+    "type": "GUNMOD",
+    "name": { "str": ".22 upper receiver" },
+    "description": "This is a complete upper receiver assembly containing a barrel, handguard, and bolt carrier group that, given the right knowledge, is installable upon an AR-15-style rifle's lower frame.  Swappable uppers enable compatible firearms to be retooled for different cartridges and barrel lengths, with this kit furnishing a weapon with a 16-inch barrel chambered for .22 rounds.",
+    "//": "M4-22 Gen II Upper 22LR/16''BBL",
+    "variant_type": "gun",
+    "weight": "2275 g",
+    "volume": "1696 ml",
+    "longest_side": "600 mm",
+    "integral_volume": "1696 ml",
+    "integral_weight": "2275 g",
+    "integral_longest_side": "406 mm",
+    "barrel_length": "406 mm",
+    "price": "284 USD 53 cent",
+    "price_postapoc": "5 USD",
+    "material": [ "steel", "plastic" ],
+    "symbol": ":",
+    "color": "dark_gray",
+    "location": "bore",
+    "mod_targets": [ "modular_ar15", "modular_m4_carbine", "modular_m16a4", "modular_m16_auto_rifle", "modular_m27_assault_rifle" ],
+    "install_time": "5 m",
+    "dispersion_modifier": 15,
+    "overwrite_min_cycle_recoil": 83,
+    "//0": "AR has too big recoil for .22 round, 83 is right on the edge of being able to use blackpowder loaded and ratshot",
+    "ammo_modifier": [ "22" ],
+    "magazine_adaptor": [ [ "22", [ "22_ar15mag_10", "22_ar15mag_28" ] ] ],
+    "pocket_mods": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "22_ar15mag_10", "22_ar15mag_28" ] } ],
+    "min_skills": [ [ "weapon", 2 ] ],
```

Comment:
```suggestion
    "min_skills": [ [ "weapon", 2 ] ],
    "damage_modifier": 2,
```
![image](https://github.com/user-attachments/assets/ced00871-a168-4ea4-b9a7-657250e707f3)
.22 LR doesn't have the special ballistic by the inch data yet, so damage modifiers need to be calculated/added manually.